### DISCUSSION TITLE
Netlify builds are constantly failing

### DISCUSSION THREAD

**Discussion by nunnatsa** [golangci/golangci-lint#3382] [Q&A] [answered]
In my PR #3369, all the unruffled-torvalds-a209db lanes are constantly failing. If I understand the log correctly, this is a test environment issue. e.g. 

> UNSUPPORTED BUILD IMAGE
>
>  The build image for this site uses Ubuntu 16.04 Xenial Xerus, which is no longer supported.
>
 > To enable builds for this site, select a later build image at the following link:
  https://app.netlify.com/sites/unruffled-torvalds-a209db/settings/deploys#build-image-selection
>
 > For more details, visit the build migration guide:
  https://answers.netlify.com/t/please-read-end-of-support-for-xenial-build-image-everything-you-need-to-know/68239

How can I make these lanes to succeed?

Thanks!

**Comment by ldez** [comment, marked as answer]
Hello,

We currently have a problem with our Netlify builds, I'm trying to fix it.
I know how to fix but I don't have the rights to edit the Netlify settings, so I'm talking with Denis, in private, to handle that.
